packages/models/src/purpose-template/purposeTemplate.ts

```diff
@@ -0,0 +1,48 @@
+import { z } from "zod";
+import {
+  DescriptorId,
+  EServiceId,
+  PurposeTemplateId,
+  TenantId,
+} from "../brandedIds.js";
+import { RiskAnalysisFormTemplate } from "../risk-analysis/riskAnalysis.js";
+import { TenantKind } from "../tenant/tenant.js";
+
+export const purposeTemplateState = {
+  draft: "Draft",
+  active: "Active",
+  suspended: "Suspended",
+  archived: "Archived",
+} as const;
+export const PurposeTemplateState = z.enum([
+  Object.values(purposeTemplateState)[0],
+  ...Object.values(purposeTemplateState).slice(1),
+]);
+export type PurposeTemplateState = z.infer<typeof PurposeTemplateState>;
+
+export const EServiceDescriptorVersionPurposeTemplate = z.object({
```

Comment:
E-service descriptor is a synonym for e-service version in our codebase, but we don't have the concept of Descriptor Version - I would name this `EServiceDescriptorPurposeTemplate` or `EServiceVersionPurposeTemplate` (the first one is better IMO, since it is coeherent with the presence of `descriptorId`.

```suggestion
export const EServiceDescriptorPurposeTemplate = z.object({
```